<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display</class>
 <widget class="QWidget" name="Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gLChart">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QSlider" name="vSVertPos">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>\</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="hSTrigHor">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="vSTrigVert">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tWChannel">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabMeas">
      <attribute name="title">
       <string>Estimation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>50%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="vLEstim">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConf">
      <attribute name="title">
       <string>Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QDial" name="dialVoltDiv">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cBOnOff">
           <property name="text">
            <string>On/Off</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cBChannel"/>
         </item>
         <item>
          <widget class="QComboBox" name="cBVoltDiv">
           <item>
            <property name="text">
             <string>2mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1V</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2V</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5V</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10V</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cBCoupling">
           <item>
            <property name="text">
             <string>DC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GND</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cBMult">
           <item>
            <property name="text">
             <string>x1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x10000</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cBVoltDiv</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dialVoltDiv</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialVoltDiv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cBVoltDiv</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
